hw6-2/cycleList.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+struct CycleListElement;
+
+struct CycleList;

```

Comment:
```suggestion
typedef struct CycleList CycleList;
```
При таком определении не нужно будет писать stuct в функциях ниже.